### DISCUSSION TITLE
GitHub responded "Git LFS is disabled for this repository"

### DISCUSSION THREAD

**Discussion by philipmortimer** [community/community#35810] [New to GitHub] [answered]
When attempting to push to a repo (git push), I get the following error:

batch response: Git LFS is disabled for this repository.
Uploading LFS objects:   0% (0/100), 0 B | 0 B/s, done.
batch response: Git LFS is disabled for this repository.
batch response: Git LFS is disabled for this repository.
error: failed to push some refs to 'https://github.com/repoName.git'

I have checked my Git LFS usage allowances and I haven't used any of it. It appears that others are accounting similar issues (specifically "https://github.com/orgs/community/discussions/23937", "https://github.com/desktop/desktop/issues/11956" and "https://github.com/git-lfs/git-lfs/issues/2481"). It appears their cases, the issue was resolved by a GitHub employee fixing some GitHub side issue. Any clue how I can fix this? (This was the easiest way I could find to contact GitHub support).

Apologies if I've said anything stupid, I'm very new to this whole programming thing.
Thanks,
Philip

**Comment by airtower-luna** [comment, marked as answer]
This is the community forum, you're mostly talking to other users (like me) here, though some staff are around, too. It's probably best if you use their [support form](https://support.github.com/request) to reach them. I hope you get it fixed soon!

**Reply by philipmortimer** [reply]
Thank you very much for your answer, that helps a lot!:)
